flink-end-to-end-tests/flink-failure-enricher-test/pom.xml

```diff
@@ -54,7 +54,13 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 			<version>${project.version}</version>
 			<scope>provided</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-connector-files</artifactId>
+			<version>${project.version}</version>
```

Comment:
nit: The formatting seems off here

Reply:
Thank you for your suggestion. I believe this is an issue with Git. The code in the master branch appears to be formatted correctly.

![image](https://github.com/user-attachments/assets/9f3a1501-2282-4789-a2ca-1a866c5a335d)


This problem was identified during the @reswqa review. However, due to the large amount of this PR, the review process on GitHub became very slow, leading to an offline review of the PR instead of using GitHub.